welcome/index.adoc

```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
+- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
 ifndef::openshift-origin[]
-- **xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Install a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}]**: You can install {product-title} with z/VM on {ibm-z-name} and {ibm-linuxone-name} on user-provisioned infrastructure.
+- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on {ibm-cloud-name}]**: On {ibm-cloud-name}, you can install {product-title} on installer-provisioned infrastructure.
 
-- **xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Install a cluster with RHEL KVM on {ibm-z-name} and {ibm-linuxone-name}]**: You can install {product-title} with RHEL KVM on {ibm-z-name} and {ibm-linuxone-name} on user-provisioned infrastructure.
-endif::openshift-origin[]
+- **xref:../installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.adoc#preparing-to-install-on-ibm-power-vs[Install a cluster on {ibm-power-name} Virtual Server]**: On {ibm-power-name} Virtual Server, you can install {product-title} on installer-provisioned infrastructure.
 
-- **xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Install a cluster on VMware vSphere]**: You can install {product-title} on supported versions of vSphere.
+- **xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[Install a cluster on {ibm-power-name}]**: On {ibm-power-name}, you can install {product-title} on user-provisioned infrastructure.
 
-- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using the bare metal user-provisioned infrastructure route.
+- **xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Install a cluster on {ibm-z-name} and {ibm-linuxone-name}]**: On {ibm-z-name} and {ibm-linuxone-name}, you can install {product-title} on user-provisioned infrastructure.
+endif::openshift-origin[]
+////
+// Addition will be tracked through https://issues.redhat.com/browse/OSDOCS-9514
+- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
```

Comment:
Consider breaking up the long sentence.
```suggestion
- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This allows you to run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
```

Reply:
"Allows" would be a form of anthropomorphism, so I'll stick with the original revision. Thanks.

Reply:
NP. How about?

> This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.

Reply:
Sounds good to me! Thanks :nerd_face: